en/flight_modes_fw/README.md

```diff
@@ -0,0 +1,173 @@
+# Flight Modes (Fixed Wing)

```

Comment:
Note, new top level Fixed wing doc. Created from the original getting started. The Overview is a list of the modes with just enough info for selection. Lower down is the high level explanation. Sidebar topic includes this plus the detailed information

Reply:
I would think about removing the middle one, and only have a rough summary and then the very detailed one.